src/components/views/elements/ReplyThread.tsx

```diff
@@ -88,7 +88,10 @@ export default class ReplyThread extends React.Component<IProps, IState> {
         // could be used here for replies as well... However, the helper
         // currently assumes the relation has a `rel_type`, which older replies
         // do not, so this block is left as-is for now.
-        const mRelatesTo = ev.getWireContent()['m.relates_to'];
+        //
+        // We're using ev.getContent() over ev.getWireContent() to make sure
+        // we grab the latest edit with potentially new relations.
+        const mRelatesTo = ev.getContent()['m.relates_to'];
```

Comment:
We probably want to also update `matrix-js-sdk` `event.getRelation()` to pull from `getContent()` as well, [`src/models/event.ts#L1077-L1087`](https://github.com/matrix-org/matrix-js-sdk/blob/df039d3d8c5241c0c85079d3fe667a6e580ac8c5/src/models/event.ts#L1077-L1087). I can create a follow-up PR after this is deemed acceptable 🙇

Reply:
Updating the js-sdk makes sense, though I'm a bit concerned that the impact could be greater than we anticipate. Until it's reported to cause bugs, we should leave it alone I think.

Reply:
I know what you mean 🤔

I feel like we may be leaving other clients in the dust if we don't change it though. Someone making a client, bridge, etc just falls into the pitfall of not supporting this use case.

Reply:
We do have a breaking change queued up for the js-sdk's next release, so squeezing it in there would be the safest bet I think.

Reply:
I guess I'll skip on adjusting it there because trying to make the change and I run into the `rel_type` and `event_id` constraint which isn't present for `m.in_repy_to` and not sure of the downstream effects.